source/documentation/deploying_services/redis.md

```diff
@@ -193,34 +195,38 @@ Type `yes` when asked for confirmation.
+
+- `tiny-3.2`
+- `medium-ha-3.2`
+
+Do not use the `tiny-clustered-3.2` service plan for new service instances as we have deprecated this plan.
 
-There are two plans available for the Redis service:
+Both service plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
 
-- `tiny-clustered-3.2`
-- `tiny-unclustered-3.2`
+Amazon ElastiCache Redis backs up both service plans every day.
 
-You cannot migrate a service from an unclustered to a clustered plan or vice versa.
+You can [vertically scale](/managing_apps.html#scaling) or upgrade your service plan.
 
-Both plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
+#### Billing
 
-#### tiny-clustered-3.2
+If your org is in its trial period, you can use the `tiny-3.2` service plan for free.
 
-We recommend that you use this plan as it is backed up every day. Note that:
+Trial orgs cannot access paid service plans by default. If paid service plans are not enabled, when you try to use a paid service plan, you will receive an error stating “service instance cannot be created because paid service plans are not allowed”.
 
-- [Sidekiq](https://sidekiq.org) [external link] does not work with clustered Redis
-- this plan cannot be [vertically scaled](/managing_apps.html#scaling) or upgraded to a bigger plan
+One of your Org Managers must contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to request that we enable paid service plans.
 
-#### tiny-unclustered-3.2
+#### High availability
 
-Use this plan if your client library cannot connect to clustered plans. Note that:
+You can use a high availability service plan (`HA`). When you use a high availability service plan, Amazon ElastiCache Redis provides a hot standby service for failover in the event that the original service instance fails.
 
-- it is not automatically backed up
-- it can be [vertically scaled](/managing_apps.html#scaling) or upgraded to a bigger plan
+If you have a non-HA service plan, you will lose data during a service instance failure. Before deciding which service plan to use, you should assess your data and what type of plan you need.
 
-Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you need an unclustered plan with backups enabled, or a larger unclustered plan.
+Refer to the [Amazon ElastiCache documentation on failover](https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/AutoFailover.html) [external link] for more information on failover.
 
-Refer to the [Redis documentation](https://redis.io/topics/cluster-tutorial) [external link] for more information on clusters.
+Refer to the [Redis documentation](https://redis.io/topics/cluster-tutorial) [external link] for more information on Redis overall.
```

Comment:
I think we need to change this link to reflect the change in the text - this is still linking to the Redis clustering information. Do we even need this link at all?

Reply:
Removed and tweaked other link to be incorporated into the text